src/components/login/LoginDetails.js

```diff
@@ -0,0 +1,293 @@
+               }
+            }
+         }
+      }
+   }
+   @media screen and (max-width: 767px) {
+      &&& {
+         .details {
+            min-height: calc(100vh - 126px);
+         }
+         .transactions {
+            padding: 0px;
+
+            > .column {
+               padding: 0px;
+            }
+         }
+      }
+   }
+`
+
+class LoginDetails extends Component {
+   render() {
+      const { contractId, transactions, fees, appTitle } = this.props
+
+      return (
+         <CustomGrid padded>
+            <Grid.Row centered className='transactions'>
+               <Grid.Column largeScreen={6} computer={8} tablet={10} mobile={16}>
+                  <div className='top-back'>
+                     <Link to='/login'>
+                        <div className='back-button h3 font-benton color-blue'>
+                           {/* <div><IconArrowLeft /></div> */}
+                           <div>Back</div>
+                        </div>
+                     </Link>
+                  </div>
+                  {contractId && (
+                     <div className='details'>
+                        <div className='details-item title h3'>Detailed description of transaction</div>
+                        <TransactionsList transactions={transactions} />
+
+                        <div className='details-item'>
+                           <div className='title h3'>
+                              Transaction Allowance
+                           </div>
+                           <div className='details-subitem font-bold color-charcoal-grey'>
+                              <div className='desc font-small'>
+                                 This app can use your NEAR for transaction fees
+                              </div>
+                              <div>
+                                 Total Allowance: .001Ⓝ
+                              </div>
+                           </div>
+                        </div>
+                        <div className='details-item'>
+                           <div className='title h3'>
+                              Transaction Fees
+                           </div>
+                           <div className='details-subitem font-bold color-charcoal-grey'>
+                              <div>Gas Limit: {fees.gasLimit}</div>
+                              <div>Gas price estimate is unavailable</div>
+                           </div>
+                        </div>
+                     </div>
+                  )}
+                  {!contractId && (
+                     <div className='details'>
+                        <div className='details-item'>
+                           <GlobalAlert 
+                              globalAlert={{
+                                 success: false,
+                                 messageCodeHeader: 'Warning',
+                                 messageCodeDescription: 'This allows access to your entire balance. Please proceed with caution.'
```

Comment:
This ```GlobalAlert``` component is cutting off a word. [Screenshot](https://drive.google.com/file/d/19oqznaWkagm59h-B80cxH2Hsi9QV7euA/view?usp=sharing)